src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -72,6 +72,25 @@ public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
             }
         }
 
+        [Fact]
+        [OuterLoop("Opens program")]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void ProcessStart_DirectoryNameInCurDirectorySameAsFileNameInExecDirectory_Success()
+        {
+            string fileToOpen = "dotnet";
+            var dotnetFolderInCurDirectory = Path.Combine(Environment.CurrentDirectory, "dotnet");
+            if (!Directory.Exists(dotnetFolderInCurDirectory))
+            {
+                Directory.CreateDirectory(dotnetFolderInCurDirectory);
+            }
+            using (var px = Process.Start(fileToOpen))
+            {
+                px.Kill();
+                px.WaitForExit();
+                Assert.True(px.HasExited);
```

Comment:
Personally I wouldn't bother with these last two lines. You don't need ot wait for it to exit. Also, you aren't testing px.HasExited and tests should not assert anything they're not testing for -- it's just extra verbiage and distraction.